packages/cli/src/commands/init/errors/DirectoryAlreadyExistsError.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+import path from 'path';
+
+export default class DirectoryAlreadyExistsError extends Error {
+  constructor(directory: string) {
```

Comment:
Shall we use this approach throughout other parts of the codebase?

Reply:
I like having one error per file. It is just my preference carried over from `Java`. Also having:
```jsx
throw new DirectoryAlreadyExistError();
```
seems more readable than
```jsx
throw new Error('This is the description why this directory should not be present');
```